cellxgene_resources/curation_sample_code.ipynb

```diff
@@ -651,6 +790,58 @@
     "**IN PROGRESS: If a reference other than one of the default CellRanger reference files was used, a map file can be created from the annotation file for use in curation**"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**If a GENCODE/ENSEMBL reference was used, parse the annotation file**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Use pd.read_table to parse the gtf & apply various manipulations to produce symbol-ENSG table\n",
+    "gtf = pd.read_table('path/to/gtf', header=None)\n",
+    "gene_dict = {\n",
+    "        'gene_ids': [],\n",
+    "        'gene_symbols': []\n",
+    "    }\n",
+    "\n",
+    "# extract gene_ids and gene_symbols from last column in the gtf table\n",
+    "for index,row in gtf.iterrows():  \n",
```

Comment:
Will be a lot more efficient to manipulate the dataframe in place rather than export that to a dictionary.
And the print statements here are excessive. There are far too many for them to be reviewable/actionable.